chalicelib/check_setup.json

```diff
@@ -487,1247 +597,1556 @@
         "schedule": {
             "monday_checks": {
                 "data": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+                    "dependencies": [],
+                    "kwargs": {
+                        "primary": true
+                    }
                 }
             }
         }
     },
-    "new_or_updated_items": {
-        "title": "New or Modified Submissions",
-        "group": "Metadata checks",
+    "check_status_mismatch": {
+        "title": "Experiment Sets containing Items with mismatch status",
+        "group": "Audit checks",
         "schedule": {
-            "morning_checks": {
-                "mastertest": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "morning_checks_2": {
+                "all_4dn": {
```

Comment:
We should be careful about scheduling checks this way, is there necessarily a need? We should consider a concept of "suppressed" schedules for (Chalice) dev and (ElasticBeanstalk) test environments, particularly for checks that run very frequently. This will save us resources over time.